doc/tools/apigen.py

```diff
@@ -231,40 +244,51 @@ def _parse_module_with_import(self, uri):
 
             functions = []
             classes = []
+            constant_variables = []
```

Comment:
If it is a constant, it is not a variable. Rename simply to `constants` (as in the docstring), and matching the `classes` pattern.